purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -278,6 +279,77 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         )
     }
 
+    /**
+     * This method will send a purchase to the RevenueCat backend. Call this when using your own implementation
+     * for subscriptions anytime a sync is needed, like after a successful purchase, or when migrating existing
+     * users to RevenueCat
+     *
+     * @warning This function should only be called if you're migrating to RevenueCat or in observer mode.
+     *
+     * @param [productID] Product ID associated to the purchase.
+     * @param [purchaseToken] Token that represents the purchase. This should be [purchaseToken] for Google purchases
+     * and the [receiptId] for Amazon purchases.
+     * @param [amazonUserID] Amazon only. Amazon's userID.
```

Comment:
could we add something saying what to do for google? it's pretty obvious since it's nullable, but just to be extra clear